### ISSUE TITLE
In case of Merge Conflicts

### ISSUE THREAD

**Issue by TechSpiritSS** [ron2111/Hacktoberfest2021_awesome-projects#126]
Before making any Pull Request always fetch your repo to the latest and after that send the Pull Request to avoid any conflicts.
Do not edit the Readme file.
Add your project info in project_doc.md after fetching the GitHub Repo

![image](https://user-images.githubusercontent.com/76687985/135597971-362cce48-ef78-473c-92dc-86f844c9fd1a.png)

**Comment by Manas1820** [comment]
I guess it won't work if you end up sending a pr but another pr was merged after you send a pr then the conflicts will persist.

**Comment by ron2111** [comment]
Yes, but it is important to update the project_doc , so the contributors should raise the PR again after fetching upstream @Manas1820

**Comment by TechSpiritSS** [comment]
None is fetching their Repo which is causing the problems with Pull Requests

**Comment by Manas1820** [comment]
In case of conflicts in the Readme for contributors list you can add this - https://github.com/marketplace/actions/auto-add-contributors

**Comment by TechSpiritSS** [comment]
that's great, @ron2111 we can use this to make readme file task easier

**Comment by Manas1820** [comment]
I have used it and seen it used many times and the UI generated is cool too
